packages/aws-cdk-lib/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -55,6 +87,32 @@ export interface AcceleratorAttributes {
    * The DNS name of the accelerator
    */
   readonly dnsName: string;
+
+  /**
+   * IP addressess associated with the accelerator.
+   */
+  readonly ipAddresses?: string[];
```

Comment:
Depending on [this comment](https://github.com/aws/aws-cdk/pull/28055/files#r1400194833), it will be removed.